types/event-map.d.ts

```diff
@@ -0,0 +1,72 @@
+declare type EventMapKey = string
```

Comment:
Are `.d.ts` files necessary when the source is written in TS?

Reply:
Shouldn't be

Reply:
I'm not 100% sure I understand what you're wanting to do but tsc only runs on typescript files so if you want it to generate typescript definition files for only those that have been moved over that should happen naturally. I'm sure we could find what you're looking for in the [compiler docs](https://www.typescriptlang.org/docs/handbook/compiler-options.html). But if you clarify what you are looking to do I could do more research.